<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-E8V8CURT" ignored="0" name="TestSelenium" tests="3" failures="1" timestamp="2021-06-07T11:48:35 EDT" time="31.274" errors="0">
  <testcase name="test2" time="12.478" classname="test.RemoteTest"/>
  <testcase name="repoOne" time="0.438" classname="test.SampleTest">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:48)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at test.SampleTest.repoOne(SampleTest.java:63)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- repoOne -->
  <testcase name="test1" time="13.275" classname="test.SampleTest"/>
</testsuite> <!-- TestSelenium -->
